tests/functional/lib/helpers.js

```diff
@@ -62,8 +63,14 @@ define([
   const takeScreenshot = thenify(function () {
     return this.parent.takeScreenshot()
       .then(function (buffer) {
-        console.error('Capturing base64 screenshot:');
-        console.error('data:image/jpeg;base64,' + buffer.toString('base64'));
+        const screenCaptureHost = 'https://screencap.co.uk';
+        return got.post(`${screenCaptureHost}/png`, { body: buffer, followRedirect: false })
+          .then((res) => {
+            console.log(`Screenshot saved at: ${screenCaptureHost}${res.headers.location}`);
+          }, (err) => {
+            console.error('Capturing base64 screenshot:');
+            console.error(`data:image/png;base64,${buffer.toString('base64')}`);
```

Comment:
I changed `image/jpeg` to `image/png` because check this out from when it was `jpeg`:


<img width="596" alt="screen shot 2017-06-12 at 15 51 53" src="https://user-images.githubusercontent.com/848085/27039940-6dd97a6a-4f87-11e7-8589-123b49f317d7.png">

In the `title` it says `PNG`